doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md

```diff
@@ -407,6 +407,9 @@ server {
 
     # Assets
     # Still use a whitelist approach to prevent each and every missing asset to go through the PHP Engine.
+    # If you are using remote storages like S3 or Google Cloud Storage, this doesn't work. You either deactivate it and handle it in PHP
+    # or redirect these suffixes directly to your CDN URL. Additionally you should configure the frontend url prefixes accordingly, see: https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/File_Storage_Setup.html
+    # location ~* ^(?!/admin)(.+?)\.((?:css|js)(?:\.map)?|jpe?g|gif?)$ {
     location ~* ^(?!/admin)(.+?)\.((?:css|js)(?:\.map)?|jpe?g|gif|png|svgz?|eps|exe|gz|zip|mp\d|m4a|ogg|ogv|webm|pdf|docx?|xlsx?|pptx?)$ {
```

Comment:
I don't get it. What is this commented `location` line for? Why does it only contain some of the types below?

Reply:
true, something went wrong there. I removed the line. Idea is, that, if you use remote storages, nginx can never resolve these files. In our Cloud Kubernetes Setup, we completely commented the location handle out.